controllers/clusters/kafka_controller.go

```diff
@@ -265,9 +267,180 @@ func (r *KafkaReconciler) handleUpdateCluster(
 		return models.ReconcileRequeue
 	}
 
+	for _, ref := range k.Spec.UserRefs {
+		err = r.handleCreateUser(ctx, k, l, ref)
+		if err != nil {
+			l.Error(err, "Cannot create Kafka user", "user", ref)
+			r.EventRecorder.Eventf(k, models.Warning, models.CreatingEvent,
+				"Cannot create user. Reason: %v", err)
+		}
+	}
+
```

Comment:
please remove it. this is already handled in predicates.

Reply:
done